forc-client-ops/src/deploy/cmd.rs

```diff
@@ -1,11 +1,9 @@
-use crate::ops::forc_deploy;
-use anyhow::{bail, Result};
 use clap::Parser;
 
-/// Deploy contract project.
-/// Crafts a contract deployment transaction then sends it to a running node.
 #[derive(Debug, Default, Parser)]
-pub struct Command {
+#[clap(version)]
+#[clap(bin_name = "forc deploy")]
```

Comment:
does this mean that we maintain the command line interface of `forc run` and `forc deploy`? And we don't have to type `forc client deploy`, right?

Reply:
Yes, `forc deploy` is sufficient 👍🏻